server/main.py

```diff
@@ -94,3 +95,40 @@ async def calculate_spectrum(payload: Payload):
                 "units": spectrum.units[payload.mode],
             },
         }
+
+
+@app.get("/download")
+async def download(payload: Payload):
+    try:
+        spectrum = radis.calc_spectrum(
+            payload.min_wavenumber_range,
+            payload.max_wavenumber_range,
+            molecule=[species.molecule for species in payload.species],
+            mole_fraction={
+                species.molecule: species.mole_fraction for species in payload.species
+            },
+            # TODO: Hard-coding "1,2,3" as the isotopologue for the time-being
+            isotope={species.molecule: "1,2,3" for species in payload.species},
+            pressure=payload.pressure,
+            Tgas=payload.tgas,
+            Tvib=payload.tvib,
+            Trot=payload.trot,
+            path_length=payload.path_length,
+            export_lines=False,
+            wstep="auto",
+            databank=payload.database,
+            use_cached=True,
+        )
+        if payload.use_simulate_slit is True:
+            spectrum.apply_slit(payload.simulate_slit, "nm")
```

Comment:
May be nice to abstract this to a function used by both download and calc_spectrum endpoints that takes in a payload and returns a spectrum object from radis.

Reply:
working on @suzil !!